app/src/App.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Redirect, Route, Switch } from 'react-router-dom';
+import Loadable from 'react-loadable';
+import TopBarProgress from 'react-topbar-progress-indicator';
+import Layout from 'components/layout';
+
+/* webpackChunkName: "SearchEngine" */
+const SearchEngine = Loadable({
+  loader: () => import('./views/search-engine'),
+  loading: TopBarProgress,
+});
+
+const App = () => {
+  return (
+    <Switch>
+      <Layout>
+        <Route exact path="/" component={SearchEngine} />
```

Comment:
Do you need the path="/", I would guess that is the default path and removing it makes no difference

Reply:
For each component/view you create, you must specify the path that references that component. Plus this is good cause if the user receives an incorrect path, they will be redirected by default to the root folder.

You can always check react-router-dom library for reference. Here is a [quick guide to get started](https://reactrouter.com/web/guides/quick-start).